SU-SWS/decanter-web
# ADAPT-4805: add routes and mdx pages

# READY FOR REVIEW

# Summary
- [ADAPT-4805](https://stanfordits.atlassian.net/browse/ADAPT-4805): Setup page routes on the Gatsby v7 site using the [Gatsby Filesystem Route API](https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/).

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Clone repo and cd into it
2. Run `npm install`
3. Run `npm run develop`
4. Confirm the site loads up and homepage is displaying (should just say "Homepage")
5. Type in a random route so that you can view the 404 page which lists all created routes/pages.
6. Confirm that the routes are set up correctly and display their page title and body (from related .mdx file)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ADAPT-4805](https://stanfordits.atlassian.net/browse/ADAPT-4805)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
